src/control/drpc/drpc_server.go

```diff
@@ -98,9 +99,9 @@ func (d *DomainSocketServer) Start(ctx context.Context) error {
 	}
 	d.listener = lis
 
-	// TODO: Should we set more granular permissions? The only writer should
-	// be the I/O Engine and we should know which user is running it.
-	if err := os.Chmod(d.sockFile, 0777); err != nil {
+	// The only writer should be the I/O Engines which should be running as the same user as
```

Comment:
Please, could you confirm that it is not needed to change the owner of the socket if the daos server is directly launch with the root account or systemd ?
From my understanding, should not be.